src/main/java/net/glowstone/chunk/GlowChunk.java

```diff
@@ -242,7 +246,7 @@ public void initializeSections(ChunkSection... initSections) {
         }
         //GlowServer.logger.log(Level.INFO, "Initializing chunk ({0},{1})", new Object[]{x, z});
 
-        sections = new ChunkSection[DEPTH / SEC_DEPTH];
+        sections = new ChunkSection[SEC_COUNT];
         System.arraycopy(initSections, 0, sections, 0, Math.min(sections.length, initSections.length));
```

Comment:
Do you think it would be beneficial to add a method to init an individual section by index?